package.json

```diff
@@ -321,6 +321,7 @@
     "@types/react-dom": "^19.0.0",
     "readable-stream": "3.6.0",
     "react-polyglot@^0.7.2": "patch:react-polyglot@npm%3A0.7.2#./.yarn/patches/react-polyglot-npm-0.7.2-636f85156f.patch",
-    "gridplus-sdk/bs58check": "2.1.2"
+    "gridplus-sdk/bs58check": "2.1.2",
+    "@shapeshiftoss/hdwallet-gridplus": "1.62.12"
```

Comment:
If that works, that'll do for now - we don't really care about the hdwallet-gridplus bump here either way, we really wanted 1.61.12 for hdwallet-trezor, 1.62.13 was published and used here just to try and unrug things.
Gives us time to do another hdwallet publish (many things in-flight there) and see if that helps.

Reply:
Update: does work with 1.62.11 (current in develop) 🎉 

@kaladinlight for viz - we'll need to revert this and figure out what got borked there